gstools/tools/export.py

```diff
@@ -53,27 +54,56 @@ def vtk_export_structured(filename, pos, fields):  # pragma: no cover
         if len(fields[field]) != len(x) * len(y) * len(z):
             raise ValueError(
                 "gstools.vtk_export_structured: "
-                + "field shape doesn't match the given mesh"
+                "field shape doesn't match the given mesh"
             )
-    gridToVTK(filename, x, y, z, pointData=fields)
+    return x, y, z, fields
 
 
-def vtk_export_unstructured(filename, pos, fields):  # pragma: no cover
+
+def to_vtk_structured(pos, fields):  # pragma: no cover
+    """Create a vtk structured rectilinear grid from a field.
+
+    Parameters
+    ----------
+    pos : :class:`list`
+        the position tuple, containing main direction and transversal
+        directions
+    fields : :class:`dict` or :class:`numpy.ndarray`
+        Structured fields to be saved.
+        Either a single numpy array as returned by SRF,
+        or a dictionary of fields with theirs names as keys.
+    """
+    x, y, z, fields = _vtk_structured_helper(pos=pos, fields=fields)
+    try:
+        import pyvista as pv
+        grid = pv.RectilinearGrid(x, y, z)
+        grid.point_arrays.update(fields)
+    except ImportError:
+        raise ImportError("Please install PyVista to create VTK datasets.")
+    return grid
+
+
+def vtk_export_structured(filename, pos, fields):  # pragma: no cover
     """Export a field to vtk structured rectilinear grid file.
 
     Parameters
     ----------
     filename : :class:`str`
         Filename of the file to be saved, including the path. Note that an
-        ending (.vtu) will be added to the name.
+        ending (.vtr) will be added to the name.
     pos : :class:`list`
         the position tuple, containing main direction and transversal
         directions
     fields : :class:`dict` or :class:`numpy.ndarray`
-        Unstructured fields to be saved.
+        Structured fields to be saved.
         Either a single numpy array as returned by SRF,
         or a dictionary of fields with theirs names as keys.
```

Comment:
Please provide a description of what is being returned.

Reply:
Nothing is returned here. it is just good practice to return the call to the function that this function wraps.